### DISCUSSION TITLE
Not allowed to use same email for reverification

### DISCUSSION THREAD

**Discussion by raf181** [community/community#63130] [GitHub Education]
### Hi everyone,

I'm trying to reverify my GitHub student privileges but it does not allow me to re-use the email and my school does not give a new email every year, also the school domain for the student email is wrong and is set to the administrative and teachers only domain

**Comment by humairshoukat** [comment]
Verify your student status with your Student ID card with clear mention of expiration date

**Reply by raf181** [reply]
we do not have a student ID, even if I had one I'm not able to submit it since I can't use my school-issued email again

**Reply by humairshoukat** [reply]
Reach your closet GitHub Campus Expert here https://education.github.com/experts
He/She can raise an internal issue for your problem.
